homework04/homework04.py

```diff
@@ -6,9 +6,9 @@
 
 if var.isdigit():
     var = int(var)
-    if bool(int(var) % 2) is False:
-         print(f'{var} is a number. It\'s even.')
-    elif bool(int(var) % 2) is True:
+    if int(var) % 2:
```

Comment:
Discard these changes because this leads to conflicts with `main` branch

Reply:
@klymenok Oleksii I'm 90% sure it's done. Could you please confirm? Thank you!